documentation/book/proc-deploying-the-topic-operator-using-the-cluster-operator.adoc

```diff
@@ -6,25 +6,40 @@
 [id='deploying-the-topic-operator-using-the-cluster-operator-{context}']
 = Deploying the Topic Operator using the Cluster Operator
 
+This procedure describes how to deploy the Topic Operator using the Cluster Operator. Alternatively, you can deploy the Topic Operator as a standalone component. For more information, see xref:deploying-the-topic-operator-standalone-str[].
+
 .Prerequisites
 
 * A running Cluster Operator
 * A `Kafka` resource to be created or updated
 
 .Procedure
 
-. Topic Operator can be included in the Entity Operator.
-Edit the `Kafka` resource ensuring it has a `Kafka.spec.entityOperator` object that configures the Entity Operator.
+. Ensure that the `Kafka.spec.entityOperator` object exists in the `Kafka` resource. This configures the Entity Operator.
++
+[source]
+----
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+...
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
```

Comment:
not sure if we should show the `userOperator` as well here, @tombentley  wdyt?

Reply:
I could highlight the topicOperator line in bold?

Reply:
It's suggested by the IBM Style Guide, so I'll give it a try.